dspace-services/src/test/resources/config/local.properties

```diff
@@ -29,3 +29,9 @@ include = included.properties
 # (We purposefully misspell DSpace and correct it in tests)
 # See DSpaceConfigurationServiceTest.testAutomaticReload()
 prop.to.auto.reload = D-space
+
+
+# This default configuration hides the dc.description.provenance field,
+# since that usually contains email addresses which ought to be kept
+# private and is mainly of interest to administrators:
+metadata.hide.dc.description.provenance = true
```

Comment:
This setting should be moved to `/dspace-api/src/test/data/dspaceFolder/config/local.cfg`  That's the local.cfg file that is used for all tests.  (I'm surprised this works to add it to `dspace-services`).

Alternatively, (and possibly the better method), this could be simply removed and simply updated dynamically in the tests that require it by doing something like:
```
@Autowired
private ConfigurationService configurationService;

... then, in method that needs it...

configurationService.setProperty("metadata.hide.dc.description.provenance", true);
```

Keep in mind, the ConfigurationService auto-reloads (resets itself to default values) *after every test*.  So, this lets you only set it to `true` in methods that need this setting, and all others will default to false.

Reply:
Actually, @benbosman , disregard all of the above. This setting can be removed as it's the *default setting already*.  See https://github.com/DSpace/DSpace/blob/master/dspace/config/dspace.cfg#L824

Reply:
This has been adjusted